index.html

```diff
@@ -9,16 +9,187 @@
+        "nut",
+        "orange",
+        "peach",
+        "quail",
+        "rabbit",
+        "strawberry",
+        "tiger",
+        "unicorn",
+        "violet",
+        "whale",
+        "xylophone",
+        "yak",
+        "zebra",
+        "airplane",
+        "ball",
+        "car",
+        "desk",
+        "eleven",
+        "flower",
+        "guitar",
+        "hat",
+        "ice",
+        "juice",
+        "kite",
+        "lemon",
+        "mouse",
+        "nose",
+        "orange",
+        "pencil",
+        "queen",
+        "rainbow",
+        "snake",
+        "tree",
+        "umbrella",
+        "violin",
+        "water",
+        "xylophone",
+        "yoyo",
+        "zebra",
+        "angel",
+        "baby",
+        "cherry",
+        "dolphin",
+        "eagle",
+        "fairy",
+        "ghost",
+        "honey",
+        "island",
+        "jacket",
+        "king",
+        "leopard",
+        "moon",
+        "nest",
+        "owl",
+        "panda",
+        "quilt",
+        "rain",
+        "sun",
+        "tulip",
+        "unicorn",
+        "vase",
+        "watermelon",
+        "xylophone",
+        "yarn",
+        "zeppelin",
+        "apple",
+        "banana",
+        "cat",
+        "dog",
+        "elephant",
+        "fish",
+        "grape",
+        "horse",
+        "ice cream",
+        "jungle",
+        "kiwi",
+        "lion",
+        "monkey",
+        "nut",
+        "orange",
+        "peach",
+        "quail",
+        "rabbit",
+        "strawberry",
+        "tiger",
+        "unicorn",
+        "violet",
+        "whale",
+        "xylophone",
+        "yak",
+        "zebra"];
 
-    <!-- 점수판 -->
+        //score 변수 선언
+        var score = 0;
+        //n=startbutton이 눌렸는지 안 눌렸는지를 구분해줄 변수
+        var n=0;
 
-    <!-- 무작위 단어 -->
+        //score 출력
+        scoreFunction()
 
-    <!-- 입력 필드 -->
-    <!-- 입력 버튼 -->
+        //랜덤 단어 출력
+        printRandomWord();
 
-    <!-- 게임 시작하기, 끝내기 버튼 -->
-
-    <script type="module" src="src/app.js"></script>
+        //게임 시작 버튼
+        function startButton(){
+            score = 0;
+            //startbutton이 눌려지면 n에 1값을 넣는다
+            n=1;
+            alert("게임이 시작되었습니다");
+        }
+        //score 출력 함수
+        function scoreFunction (){
+            document.getElementById("score").innerHTML = score;
+        }
+        //랜덤 단어 설정
+        function randomValueFromWords(_words){
+            const random = Math.floor(Math.random()*_words.length);
+            return _words[random];
+        }
+        //랜덤 단어 출력 함수
+        function printRandomWord(){
+            const word = randomValueFromWords(_words);
+            document.getElementById("word").innerHTML=word;
+        }
+        //단어 입력 함수
+        function inputAnswer(){
+            const word = document.getElementById("word").innerHTML;
+            const myanswer = document.getElementById('myanswer').value;
+            //n이 1일때만 실행
+            if(n==1){
```

Comment:
```suggestion
            if(n===1){
```

이부분만 동등연산자인데 이유가 따로 있는건가요?